package.json

```diff
@@ -21,13 +21,14 @@
     "yargs": "^16.2.0"
   },
   "scripts": {
+    "build": "babel src -d dist",
+    "dev": "nodemon --exec babel-node src/index.js",
+    "start": "npm run build && node dist/index.js",
```

Comment:
question: This is a very welcome change :star2:, as it will allow us to add debuggers and breakpoints easier when debugging the headless wallet.

However, is it possible for us to add `.map` files? Otherwise we'll be debugging and receiving stack traces from transpiled code only. This may make the development process slower.

Reply:
Done, but building sourceMaps by configuring it on the .babelrc file was only introduced in [v7.19.3](https://github.com/babel/babel/releases/tag/v7.19.3) and we are using 7.13, we should upgrade the dependencies, but I don't think this PR should cover this.

To build sourceMaps on v7.13 we can only do it by explicitly passing the `--source-maps` argument to babel cli unless we want the sourceMaps to be inline on the file but this option is not recommended for production environments.